src/frontend.md

```diff
@@ -5,31 +5,55 @@
 ```
 src/
   components/ (can be dumb and smart)
-    bankAccount/
+    [All components should be placed in some entity folder or shared folder]
+    bankAccount/ (entity, module, .etc)
       BankAccountTable/
         [no nested directories]
+        [avoid index.ts]
+        [other files should be placed in queries or services]
```

Comment:
i guess you meant not files, but hooks here? if so, could you move closed to hooks?

Reply:
so could you clarify it in code, not in PR comment?

Reply:
let's clarify in code